deploy/selfhost/install.sh

```
@@ -335,6 +391,34 @@
     startServices
 }
 function upgrade() {
+    local latest_release=$(checkLatestRelease)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)